src/lib/components/EventMap/index.js

```diff
@@ -96,7 +96,7 @@ class EventMap extends BaseStateElement {
     const date = new Date(entry.date);
     return html`
             <li>
```

Comment:
I think these items are supposed to be left aligned on mobile

![image](https://user-images.githubusercontent.com/1066253/85341529-34fbbe00-b49d-11ea-9032-5807455f21e1.png)